### DISCUSSION TITLE
Matrix chatroom for Devs

### DISCUSSION THREAD

**Discussion by GuillaumeLagrange** [pwntester/octo.nvim#620] [General]
Hey!

I think it's always nice to have a medium to have informal conversations about projects, and I took the liberty of following neovim and lazyvim's example and created a matrix chatroom.

Feel free to join and chat, and @pwntester let me know if you are against the idea 🤓

https://matrix.to/#/%23octo.nvim%3Amatrix.org

Cheers

**Comment by williambdean** [comment]
Do you have a link? How about discord or slack?

**Reply by GuillaumeLagrange** [reply]
I cannot believe I managed to forget the link in my post, I added it :sweat: 

Regarding Discord or Slack, they are also viable options, but I personally like the IRC-like aspect of matrix + more choice in apps and protocol being open source meaning potentially not relying on yet another private company, which fits open source better IMO.

If you guys are not up for Matrix I'd say I prefer discord over Slack as an alternative, In my head slack means work 😬

**Reply by pwntester** [reply]
Thanks for setting that up. Im ok using Matrix or Discord. Lets use Matrix for now and see how it works for us
